guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/social/NewProjectEventAdapter.java

```diff
@@ -112,7 +108,7 @@ public SocialActivitiesEvent toSocial(Object object) {
         return new ArrayList<String>();
     }
 
-    private String getAdditionalInfo(NewProjectEvent event) {
+    private String getAdditionalInfo( NewModuleEvent event ) {
```

Comment:
Seems like you're introducing the old code style with extra spaces between parenthesis? Could you please preserve the new formatting?

Reply:
Sure